lib/index.js

```diff
@@ -90,6 +92,7 @@ function MapboxGeocoder(options) {
   this.fresh = true;
   this.lastSelected = null;
   this.geolocation = new Geolocation();
+  return this.options.useGeocoderV6 ? new MapboxGeocoderV6(options) : this;
```

Comment:
@mpothier  Do we want to use v6 functionality via  useGeocoderV6 flag in scope of the old MapboxGeocoder, or we should consider exposing another class MapboxGeocoderV6?